metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "praekeltfoundation-seed_stack",
-  "version": "0.6.2-dev",
+  "version": "0.6.3-dev",
```

Comment:
The version is "pre-bumped". i.e. 0.6.1 is the current latest version. We remove the `-dev` before publishing a new version to puppet forge.